src/RequestModelFactory.php

```diff
@@ -88,11 +93,23 @@ private function getRequestData(ServerRequestInterface $request): array
         ];
     }
 
-    private function validateRequest(ValidatableModelInterface $model, array $requestData): void
+    private function createValidator(ValidatableModelInterface $model): Validator
+    {
+        return new Validator($model->getRules());
+    }
+
+    private function getErrorsFromValidationResult(ResultSet $result): array
```

Comment:
I suggest to move this method to constuctor `RequestValidationException`

Reply:
The exception class wants an array of errors, not something that needs to be formatted

Reply:
It can be modified :)

Reply:
in the current implementation, this does not make sense, since you still need to pull out all the errors to determine what they are.

Reply:
May be create method for get not empty errors in `ResultSet` ?